backend/tests/exploregens.spec.ts

```diff
@@ -353,7 +369,7 @@ describe('exploregens unit test', () => {
         });
 
         it("recommended array is undefined", () => {
-            workspaceConfigMock.expects("get").withExactArgs(exploregens["SEARCH_QUERY"]).returns();
+            workspaceConfigMock.expects("get").withExactArgs(exploregens["SEARCH_QUERY"]).returns("");
```

Comment:
it is better to remove .returns("") or .returns() if you do not use the returned value